rollbar/__init__.py

```diff
@@ -779,76 +780,52 @@ def _add_locals_data(data, exc_info):
             frame_num += 1
             continue
 
-        # Create placeholders for args/kwargs/locals
-        args = []
-        kw = {}
+        # Create placeholders for argspec/varargspec/keywordspec/locals
+        argspec = []
+        varargspec = None
+        keywordspec = None
         _locals = {}
```

Comment:
Good question! Remnant from before the refactor. Let me see if there's any reason we'd do this.